src/styles/index.scss

```diff
@@ -1,3 +1,123 @@
+@import './starsgit commig';
```

Comment:
There seems to be a typo in the import statement for the `stars` block. It should be `@import './stars';` instead of `@import './starsgit commig';`. Please correct this to ensure the styles are imported correctly.